CHANGELOG.md

```diff
@@ -27,6 +28,8 @@
 
 ## Next
```

Comment:
No need to change now, but I've traditionally used "Next" during development and not in releases.